streaming/src/main/scala/org/apache/spark/streaming/receiver/RateLimiter.scala

```diff
@@ -61,4 +62,21 @@ private[receiver] abstract class RateLimiter(conf: SparkConf) extends Logging {
         rateLimiter.setRate(newRate)
       }
     }
+
+  /**
+   * Get the initial rateLimit to initial rateLimiter
+   * @return
+   */
+  def getInitialRateLimit() : Long = {
+    if (RateController.isBackPressureEnabled(conf)) {
+      val initialRate = conf.getLong("spark.streaming.backpressure.initialRate", 1000)
```

Comment:
The default value should be `maxRateLimit`. And here you can use one line code, such as ```math.max(conf.getLong("spark.streaming.backpressure.initialRate", maxRateLimit), maxRateLimit)```